### ISSUE TITLE
Bug: unknown location of exported archive

### ISSUE THREAD

**Issue by Dinock** [mattermost/focalboard#2326]
## Steps to reproduce the behavior

1. Go to ... settings
2. Select  ... export archive
3. See error: There is no prompt or dialogue which offers the opportunity to export the archive to a specific location.

## Expected behavior
I would expect a dialogue to open which would make it possible to export the archive at a location chosen by the user. Therefore it is not clear where the archive has been exported to and if the export has been successful. Furthermore it is not clear what the filename of the exported archive actually is, because there is no option to (re)name it before exporting.

## Edition and Platform

 - Edition: Personal Desktop
 - Version: v0.14.0 RC3
 - Browser and OS: Firefox 97 on Linux Ubuntu 21.10

## Additional context

Kind of related to #2111 because the problem is currently not documented in the user's guide.

* Sev/2: Issue affects major functionality with a difficult or non-obvious workaround

**Comment by wuwinson** [comment]
Looks to be happening on the latest Personal Desktop version.

**Comment by chenilim** [comment]
Followup: I filed a separate issue for export not working on Mac Personal Desktop (#2333), but I believe that is unrelated to this issue - it sounds like export is working on Linux Personal Desktop, but the user is unable to select a location. Is that right @Dinock? You had also mentioned "Firefox 97", but that should not be related right?

**Comment by Dinock** [comment]
Yes, it does work and it is exported to the "downloads"-folder but one has to figure it out by guessing, because there is no prompt telling you.
Yes, this is unrelated. I just mentioned it because the GitHub-Issue-template asked for the version of my browser.
